vault/mount.go

```diff
@@ -185,6 +193,7 @@ type MountConfig struct {
 	PluginName               string        `json:"plugin_name,omitempty" structs:"plugin_name,omitempty" mapstructure:"plugin_name"`
 	AuditNonHMACRequestKeys  []string      `json:"audit_non_hmac_request_keys,omitempty" structs:"audit_non_hmac_request_keys" mapstructure:"audit_non_hmac_request_keys"`
 	AuditNonHMACResponseKeys []string      `json:"audit_non_hmac_response_keys,omitempty" structs:"audit_non_hmac_response_keys" mapstructure:"audit_non_hmac_response_keys"`
+	ListingVisibility        string        `json:"listing_visibility,omitempty" structs:"listing_visibility" mapstructure:"listing_visibility"`
```

Comment:
Should this be type `ListingVisiblityType`

Reply:
Yea, I haven’t switched over yet since the comparison logic will change depending on whether I end up using string or int for the actual type.